assets/js/util/index.js

```diff
@@ -207,17 +207,21 @@ export const numberFormat = ( number, options = {} ) => {
 /**
  * Gets the current locale for use with browser APIs.
  *
+ * @param {Object} _global The global window object.
+ *
  * @return {string} Current Site Kit locale if set, otherwise the current language set by the browser.
  *                  E.g. `en-US` or `de-DE`
  */
-export const getLocale = () => {
-	const siteKitLocale = get( global, [ 'googlesitekit', 'locale', '', 'lang' ] );
-
+export const getLocale = ( _global = global ) => {
+	const siteKitLocale = get( _global, [ 'googlesitekit', 'locale', '', 'lang' ] );
 	if ( siteKitLocale ) {
-		return siteKitLocale.replace( '_', '-' );
+		const matches = siteKitLocale.match( /(\w{2})?(_)?(\w{2})/ );
+		if ( matches[ 0 ] ) {
```

Comment:
This needs to be `matches && matches[ 0 ]` since if there is no match, this would cause an error otherwise.